### DISCUSSION TITLE
Incorrect information!!!

### DISCUSSION THREAD

**Discussion by TwilaRichards** [gc-da11yn/gc-da11yn.github.io#623] [Content (Open Discussion)]
There is a lot of incorrect information on this site.

**Comment by JulieBB23** [comment]
Hi @TwilaRichards,

Thank you for bringing this to our attention. We are continually working to improve the toolkit, and we strive to keep it as accurate and relevant as possible. If you could share specific areas where you've identified incorrect information, along with any suggestions for improvement, we would greatly appreciate it. Our content working group task force will review and address them promptly.

Your feedback is invaluable to us, and we're grateful for your contribution. Thank you for reaching out.

Best regards,

**Comment by TwilaRichards** [comment]
I am neurodivergent.
Please provide a phone number for us to discuss this matter further

**Comment by JulieBB23** [comment]
Hi,
Could you send me an email ***@***.******@***.***> and then we can schedule sometime to discuss.
Looking for to hear from you,

Julie BB

**Comment by shawnthompson** [comment]
@JulieBB23 it looks like GitHub censored your email address. 

@TwilaRichards you can find our email address on our [Contact us](https://a11y.canada.ca/en/contact-us/) page and email us there. We look forward in discussing your concerns. 

Sorry for the extra clicks.

**Comment by TwilaRichards** [comment]
***@***.***

**Reply by shawnthompson** [reply]
Sorry @TwilaRichards but Github hides all email addresses posted in these comments so we can't see your email address. You will have to visit our [Contact us](https://a11y.canada.ca/en/contact-us/) to find our email address.
